src/Sema.zig

```diff
@@ -17658,22 +17661,23 @@ fn zirTypeInfo(sema: *Sema, block: *Block, inst: Zir.Inst.Index) CompileError!Ai
                                 // TODO: write something like getCoercedInts to avoid needing to dupe
                                 const bytes = if (tuple.names.len != 0)
                                     // https://github.com/ziglang/zig/issues/15709
```

Comment:
Can this comment be removed? The issue was closed in May last year.

Reply:
Yes, it can. But maybe it will be nicer if I just merge your PR for now since it's already green.